service-mesh/app-src/apps/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if [ -z $1 ]; then
+    docker_repo=$DOCKER_REPO
+else
+    docker_repo=$1
```

Comment:
is this path ever used or advertised?

Reply:
Added it for easier testing, forgot to remove it. Thanks